docs/src/gui/qtvcp.adoc

```diff
@@ -277,12 +277,12 @@ Here we show how to add new functions and override existing ones.
 # reference: https://ruivieira.dev/python-monkey-patching-for-readability.html
 import types
 
-# import the handlerfile to get reference to it's libraries.
+# import the handlerfile to get reference to its libraries.
 # use <screenname>_handler
 import qtdragon_handler as hdlr
 
 # This is actually an unbounded function with 'obj' as a parameter.
-# You call this function without the usual preceding 'self.'
+# You call this function without the usual preceding 'self'.
```

Comment:
'self.' Is correct

Reply:
Ah. I had obviously misinterpreted that period. May I suggest to change it to 
```
'self.'.
```
?

Reply:
Ya I almost issued it too :) . Sure that looks good to me. Thanks

Reply:
Almost missed it I mean.